share/spice/time/time.js

```diff
@@ -69,56 +69,45 @@
         if(!chosen) {
             return Spice.failed('time');
         }
+        
+        DDG.require('moment.js', function(){
+            var dateObj = moment.parseZone(chosen.time.iso, moment.ISO_8601);
+            
+            // holds display location
+            var placeName = chosen.geo.state ? (chosen.geo.name + ", " + chosen.geo.state) : chosen.geo.name;
+            if (chosen.overridePlaceName) {
+                placeName = chosen.overridePlaceName;
+            }
 
-        var dateObj = DDG.getDateFromString(chosen.time.iso),
-            months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
-            days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
-
-        //Convert 24 hour time to 12 hour time
-        function toPrettyTime(date) {
-            var hours = date.getHours(),
-                minutes = date.getMinutes();
-            return {
-                hours: (hours % 12) || 12,
-                minutes: minutes < 10 ? '0' + minutes : minutes,
-                amPM: hours >= 12 ? "PM" : "AM"
+            var dateTime = {
+                time: dateObj.format("LT"),
```

Comment:
@jagtalon the use of "LT" is not needed here. See [comment](https://github.com/duckduckgo/zeroclickinfo-spice/issues/2372#issuecomment-170736182) for details.